pallets/phala/src/compute/base_pool.rs

```diff
@@ -71,6 +71,7 @@ pub mod pallet {
 	{
 		type RuntimeEvent: From<Event<Self>> + IsType<<Self as frame_system::Config>::RuntimeEvent>;
 		type MigrationAccountId: Get<Self::AccountId>;
+		type WPhaMinBalance: Get<BalanceOf<Self>>;
```

Comment:
Please call it WPhaExistentialDeposit. However, no need to change it now. We will go with the current code, and change it later when we have time.